lms/static/js/student_account/views/LoginView.js

```diff
@@ -35,20 +36,21 @@
             },
 
             render: function( html ) {
-                var fields = html || '';
-
-                $(this.el).html(_.template(this.tpl)({
-                    // We pass the context object to the template so that
-                    // we can perform variable interpolation using sprintf
-                    context: {
-                        fields: fields,
-                        currentProvider: this.currentProvider,
-                        errorMessage: this.errorMessage,
-                        providers: this.providers,
-                        hasSecondaryProviders: this.hasSecondaryProviders,
-                        platformName: this.platformName
-                    }
-                }));
+                var fields = html || '',
```

Comment:
Nit: change this to fieldsHtml and have it perform the ``HtmlUtils.HTML`` wrapping here.